dali/test/python/test_operator_python_function.py

```diff
@@ -256,6 +267,12 @@ def test_python_operator_invalid_function():
         return
     raise Exception('Should not pass')
 
+def test_python_operator_invalid_pipeline():
+    invalid_pipe = PythonOperatorInvalidPipeline(BATCH_SIZE, NUM_WORKERS, DEVICE_ID, SEED,
+                                                 images_dir, invalid_function)
```

Comment:
In this test you should pass a valid function so we know that pipeline construction fails because of the graph definition, not the actual operators inside it.

Reply:
Done. I did it, but mixed the call in the tests.